### ISSUE TITLE
mobx peerDependency incorrect

### ISSUE THREAD

**Issue by neoel** [mobxjs/mobx-utils#123]
I've installed mobx@4.1.1 with mobx-utils@4.0.0 but I get a warning that the peer dependency is incorrect. 
Update the peerDependency so it's searches _all_ valid versions of mobx.

```
warning " > mobx-utils@4.0.0" has incorrect peer dependency "mobx@4.0.0".
```

**Comment by Strate** [comment]
@mweststrate sorry for disturbing, but I would like to ping you here. Fix already merged, but not released to npm, could you release pls?

**Comment by mweststrate** [comment]
Fix has been released as 4.0.1
